spec/mongo/database_spec.rb

```diff
@@ -153,7 +153,80 @@
         expect(collection_names).to include('0_dalmatians')
         expect(collection_names).to include('1_dalmatians')
       end
+    end
+
+    context 'when provided a filter' do
+
+      before do
+        database[:users2].drop
```

Comment:
We should generally be using strings for database and collection names, so I think this should be spelled `        database['users2'].drop`.